Source/Core/VideoCommon/TextureConfig.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoCommon/TextureConfig.h"
+
+#include <tuple>
+
+bool TextureConfig::operator==(const TextureConfig& o) const
```

Comment:
No, nothing like that.  I tend to prefer forward declaring things and reducing include files in headers.  That's why this was moved to the cpp.  I usually only put simple getters/setters in the header.

I didn't think of the performance implications of this, I wonder how heavy a change this is?